app/views/errors/unprocessable_entity.html.erb

```diff
@@ -1,14 +1,15 @@
 <%= content_for :page_title, "Sorry, there’s a problem with the service" %>
```

Comment:
Should the text in this file be in a locale?

Reply:
Addressed in https://github.com/DFE-Digital/itt-mentor-services/pull/139/commits/91ee412b43ad855c42892dc57bd5f475bf809fa5